py/running_servers.py

```
@@ -535,5 +632,110 @@ def suggest(
+            "[red]✗[/red] Command uses {livereload} but no --livereload port was given"
+        )
+        raise typer.Exit(code=2)
+
+    url = get_url(chosen_http, hostname)
+    err_console.print(f"🔨 serving [cyan]{directory}[/cyan] on :{chosen_http} → {url}")
+
+    if dry_run:
+        print(" ".join(resolved))
+        raise typer.Exit(code=0)
+
+    os.execvp(resolved[0], resolved)
```

Ruff (0.15.15)

[error] Starting a process without a shell

(S606)